assets/javascripts/src/actions.es6

```diff
@@ -46,6 +46,7 @@ export function submitPayment(dispatch) {
             if (response.response.ok) dispatch({ type: PAYMENT_COMPLETE, response: response.json })
             else dispatch({ type: PAYMENT_ERROR, kind: 'card', error: response.json })
         })
+        .then(trackPayment(state.card.amount, state.data.currency.code))
```

Comment:
the `PAYMENT_COMPLETE` dispatch above will update the browser address, that might happen before the GA event goes out. I'd move `trackPayment` to happen before the dispatch